core/modules/menu/js/menu.admin.js

```diff
@@ -19,6 +19,60 @@ Backdrop.behaviors.menuAdminFieldsetSummaries = {
   }
 };
 
+Backdrop.behaviors.menu_edit_item_parents = {
+  attach: function (context, settings) {
```

Comment:
This needs a guard class or the use of `$.once()` to prevent double-binding after AJAX requests.

Reply:
I've added `.once`  but I've never quite understood what it does, so this might not be what you mean.

Reply:
Thanks, that should do the job!

If an AJAX request occurs on the page, then this `.attach()` method will get called again, and potentially re-bind the same events multiple times. We just ran into something similar with CKEditor here: https://github.com/backdrop/backdrop-issues/issues/6774.